### DISCUSSION TITLE
Hide Scrollbars?

### DISCUSSION THREAD

**Discussion by TiloNolte** [xournalpp/xournalpp#5444] [Q&A] [answered]
Just recently discovered Xournal++ and love it so far.  Imho much better then a lot of paid applications. 
But is there a way to permanently make the scrollbars on the right and bottom end of the screen go away? I often find myself accidentally touching them while drawing and lifting my pen, which leads to skipping to a completely different page and is quite annoying.
If not, can I somehow move the right bar to the left side so I don't touch while writing with my right hand?
Cheers everyone

**Comment by bhennion** [comment, marked as answer]
The view tab in the settings dialog has all the options you ask for (hide scrollbars or put them on the other side).
![Screenshot_20240110_172943](https://github.com/xournalpp/xournalpp/assets/17818041/65952632-f5d0-4038-863d-25d7f49eb2d3)

**Reply by TiloNolte** [reply]
Thanks so much! I hadn't found the feature, because the German translation is not very straightforward, or lets say nobody used it. Translated word for word it would mean like "bar that runs along while you run down the pages" or "page running down bar" but I guess the alternative would be using the english word...

**Reply by vinyx1** [reply]
BTW, the settings dialog is under **Edit -> Preferences**
